nixos/modules/services/web-apps/grocy.nix

```diff
@@ -145,7 +145,7 @@ in {
     services.nginx = {
       enable = true;
       virtualHosts."${cfg.hostName}" = mkMerge [
-        { root = "${cfg.package}/public";
+        { root = "${cfg.package}/share/php/grocy/public";
```

Comment:
This feels like a breaking change for anyone not using the official module (i.e., like me and using another reverse proxy manually).
Perhaps it should be added to the release notes (and for it to be included into 24.04, it should be merged in two days, if I understand correctly; see #303285)

I'd love the opinion of more experienced Nixpkgs members on this.

Reply:
>  it should be merged in two days, if I understand correctly; see #303285)

quite sure that grocy is not [release critical](https://nixos.github.io/release-wiki/Release-Critical-Packages.html), so should be okay till 1st of may then.

Reply:
@Shawn8901 I interpreted that line of the schedule as "only release critical packages can have breaking changes now", which obviously doesn't make any sense 😅
Thank you for clarifying! It seems like we have some more time to get this ready then :)

Reply:
I adjusted the `installPhase` to avoid the change.